examples/example-initial-signin.py

```diff
@@ -26,6 +26,7 @@ def do_initial_signin():
                 email_address=my_email_address,
                 secret_key=my_secret_key,
                 password=my_password)
+        return op
```

Comment:
Good catch.

I'd rather have the return at the bottom, and have the exception handler bail early if there's a problem.